### PR TITLE
fix: celery confused appname

### PR DESCRIPTION
**Pull request by eeliu**
- fix celery unknow node
- fix isdocker setting

### CODE REVIEW

File: collector-agent/CollectorAgent/GrpcAgent.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class GrpcAgent(GrpcClient):
     PINGID = 0
 
     def __init__(self, hostname, ip, ports, pid, address, server_type=PHP, meta=None, get_req_stat=None,
-                 maxPending=-1, timeout=300, isContainer=False):
+                 maxPending=-1, timeout=30, isContainer=False):
```
Comment (eeliu) [inline collector-agent/CollectorAgent/GrpcAgent.py:41]: reconnection waiting time is 30s

File: collector-agent/CollectorAgent/GrpcAgentImplement.py [hunk 2]
```diff
@@ -102,7 +102,7 @@ def processMain(self):
                                                          self.max_pending_sz)
         self.agent_client = GrpcAgent(agentHost.hostname, agentHost.ip, self.web_port, os.getpid(),
                                       self.agent_addr, self.service_type, self.agent_meta, self.getReqStat,
-                                      self.isContainer)
+                                      isContainer=self.isContainer)
```
Comment (eeliu) [inline collector-agent/CollectorAgent/GrpcAgentImplement.py:105]: docker: configuration file and env.file

File: collector-agent/conf/collector.conf [hunk 3]
```diff
@@ -10,10 +10,12 @@ collector.grpc.span.port=9993
 Web_Port=8001
 Log_Level=ERROR
 LOG_DIR=/tmp/
+Isdocker = false
 [Agent]
 # the same as below "pinpoint_php.CollectorHost"
 # use tcp port
 Address=0.0.0.0@9999
+collector.agent.isdocker = true
```
Comment (eeliu) [inline collector-agent/conf/collector.conf:18]: configuration file format

File: plugins/PY/pinpoint/common/Span.py [hunk 4]
```diff
@@ -63,18 +63,18 @@ def generatePinpointHeader(host,headers):
...
+    headers[PP_HEADER_PINPOINT_PAPPNAME] = pinpointPy.get_context_key(PP_APP_NAME)
     headers['Pinpoint-Flags'] = "0"
-    headers[PP_HEADER_PINPOINT_HOST] = local_host_name
+    headers[PP_HEADER_PINPOINT_HOST] = func
     headers[PP_HEADER_PINPOINT_CLIENT] = local_ip
     headers[PP_HEADER_PINPOINT_TRACEID] = pinpointPy.get_context_key(PP_TRANSCATION_ID)
     headers[PP_HEADER_PINPOINT_PSPANID] = pinpointPy.get_context_key(PP_SPAN_ID)
-    headers[PP_HEADER_PINPOINT_SPANID] =  pinpointPy.get_context_key(PP_NEXT_SPAN_ID) #nsid
\ No newline at end of file
+    headers[PP_HEADER_PINPOINT_SPANID] = pinpointPy.get_context_key(PP_NEXT_SPAN_ID)
```
Comment (eeliu) [inline plugins/PY/pinpoint/common/Span.py:80]: other framework is broken!

File: plugins/PY/pinpoint/libs/celeryCaller/TaskPlugin.py [hunk 5]
```diff
@@ -30,7 +30,7 @@ class TaskPlugin(PinpointCommonPlugin):
     def onBefore(self, *args, **kwargs):
         super().onBefore(*args, **kwargs)
         pinpointPy.add_clue(PP_SERVER_TYPE, PP_REMOTE_METHOD)
-        pinpointPy.add_clue(PP_DESTINATION,local_host_name)
+        pinpointPy.add_clue(PP_DESTINATION,args[0].name)
```
Comment (eeliu) [inline plugins/PY/pinpoint/libs/celeryCaller/TaskPlugin.py:33]: ## appid and appname

> There is no need to set appid and name in celery
name are "celery-task-queue"
appid is "task.name"

**Review by EyelynSu — APPROVED** [review]
LGTM

**Pull request merged**